x-pack/legacy/plugins/siem/public/containers/network_top_countries/index.tsx

```diff
@@ -0,0 +1,156 @@
+  topCountriesSort: NetworkTopNFlowSortField;
+}
+
+type NetworkTopCountriesProps = OwnProps & NetworkTopCountriesComponentReduxProps;
+
+class NetworkTopCountriesComponentQuery extends QueryTemplatePaginated<
+  NetworkTopCountriesProps,
+  GetNetworkTopCountriesQuery.Query,
+  GetNetworkTopCountriesQuery.Variables
+> {
+  public render() {
+    const {
+      activePage,
+      children,
+      endDate,
+      flowTarget,
+      filterQuery,
+      id = `${ID}-${flowTarget}`,
+      ip,
+      isInspected,
+      limit,
+      skip,
+      sourceId,
+      startDate,
+      topCountriesSort,
+    } = this.props;
+    const variables: GetNetworkTopCountriesQuery.Variables = {
+      defaultIndex: chrome.getUiSettingsClient().get(DEFAULT_INDEX_KEY),
+      filterQuery: createFilter(filterQuery),
+      flowTarget,
+      inspect: isInspected,
+      ip,
+      pagination: generateTablePaginationOptions(activePage, limit),
+      sort: topCountriesSort,
+      sourceId,
+      timerange: {
+        interval: '12h',
+        from: startDate!,
+        to: endDate!,
+      },
+    };
+    return (
+      <Query<GetNetworkTopCountriesQuery.Query, GetNetworkTopCountriesQuery.Variables>
+        fetchPolicy={getDefaultFetchPolicy()}
+        notifyOnNetworkStatusChange
+        query={networkTopCountriesQuery}
+        skip={skip}
+        variables={variables}
+      >
+        {({ data, loading, fetchMore, networkStatus, refetch }) => {
+          const networkTopCountries = getOr([], `source.NetworkTopCountries.edges`, data);
+          this.setFetchMore(fetchMore);
+          this.setFetchMoreOptions((newActivePage: number) => ({
+            variables: {
+              pagination: generateTablePaginationOptions(newActivePage, limit),
+            },
+            updateQuery: (prev, { fetchMoreResult }) => {
+              if (!fetchMoreResult) {
+                return prev;
+              }
+              return {
+                ...fetchMoreResult,
+                source: {
+                  ...fetchMoreResult.source,
+                  NetworkTopCountries: {
+                    ...fetchMoreResult.source.NetworkTopCountries,
+                    edges: [...fetchMoreResult.source.NetworkTopCountries.edges],
+                  },
+                },
+              };
+            },
+          }));
+          const isLoading = this.isItAValidLoading(loading, variables, networkStatus);
+          return children({
+            id,
+            inspect: getOr(null, 'source.NetworkTopCountries.inspect', data),
+            isInspected,
+            loading: isLoading,
+            loadPage: this.wrappedLoadMore,
+            networkTopCountries,
+            pageInfo: getOr({}, 'source.NetworkTopCountries.pageInfo', data),
+            refetch: this.memoizedRefetchQuery(variables, limit, refetch),
+            totalCount: getOr(-1, 'source.NetworkTopCountries.totalCount', data),
+          });
+        }}
+      </Query>
+    );
+  }
+}
+
```

Comment:
```suggestion
NetworkTopCountriesComponentQuery.displayName = 'NetworkTopCountriesComponentQuery';
```

`displayName` appears to be missing from `network_dns` [as well](https://github.com/elastic/kibana/blob/f5e91da99cf4c430e8ff63d92742645fbf7d9cec/x-pack/legacy/plugins/siem/public/containers/network_dns/index.tsx#L137-L139).

Reply:
its a class here, not a memo